package-lock.json

```diff
@@ -1,8 +1,11555 @@
 {
   "name": "sm64js",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Do you use a new version of npm? What's your Node version?

Reply:
My Node version is v16.3.0, and if I checked correctly, my npm version is 7.15.1.

All the fonts are fanmade and not official resources from Nintendo. The creators of these fonts also appear on the Special Thanks part of the credits. I've added some comments just in case too to clarify. I've cleaned up the files to remove anything that wasn't used, the unused font was made by GlitchyPSI (https://github.com/GlitchyPSIX/UltraMario66/tree/master/rel).

The textures are also either custom-made by me or made using Render96's as a base except the buttons which I changed to to use Render96's ones just in case (the stone one already used fanmade textures for the middle part but was updated to remove any trace of official assets). 

GamePainter helped with the gameCanvas border texture too.
I'll see if I can do anything with the game and chat margins but I'm not very sure either. If I'm able to find a way to make it better I'll push it too later.